modules/services/model/schemas/au.com.shiftyjelly.pocketcasts.models.db.AppDatabase/79.json

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
Nitpick: Let's highlight such files if the PR size is unexpectedly high.
This JSON (~1500 lines) is autogenerated while migrating the DB to add a new field: `image_url`.